com.unity.perception/Documentation~/Tutorial/Phase1.md

```diff
@@ -318,20 +318,20 @@ If you run the simulation now you will see the generated backgrounds look much m
 <img src="Images/background_good.png" width = "700"/>
 </p>
 
-It is now time to spawn and randomize our foregournd objects. We are getting close to generating our first set of synthetic data!
+It is now time to spawn and randomize our foreground objects. We are getting close to generating our first set of synthetic data!
 
 * **Action**: Add `ForegroundObjectPlacementRandomizer` to your list of Randomizers. Click _**Add Folder**_ and select `Assets/Samples/Perception/0.5.0-preview.1/Tutorial Files/Foreground Objects/Phase 1/Prefabs`.
 * **Action**: Set these values for the above Randomizer: `Depth = 3, Separation Distance = 1.5, Placement Area = (5,5)`.
 
-This Randomizer uses the same algorithm as the one we used for backgrounds; however, it is defined in a separate C# class because you can only have **one of each type of Randomizer added to your Scenario**. Therefore, this is our way of differentating between how background and foreground objects are treated.
+This Randomizer uses the same algorithm as the one we used for backgrounds; however, it is defined in a separate C# class because you can only have **one of each type of Randomizer added to your Scenario**. Therefore, this is our way of differentiating between how background and foreground objects are treated.
```

Comment:
Maybe remove "between"?

"Therefore, this is our way of differentiating how background and foreground objects are treated."